addons/crm/security/ir.model.access.csv

```diff
@@ -23,3 +23,4 @@ access_calendar_event_type_sale_salesman,calendar.event.type.salesman,calendar.m
 access_mail_activity_type_sale_manager,mail.activity.type.sale.manager,mail.model_mail_activity_type,sales_team.group_sale_manager,1,1,1,1
 access_crm_lead_scoring_frequency,access_crm_lead_scoring_frequency,model_crm_lead_scoring_frequency,sales_team.group_sale_salesman,1,0,0,0
 access_crm_lead_scoring_frequency_field,access_crm_lead_scoring_frequency_field,model_crm_lead_scoring_frequency_field,sales_team.group_sale_salesman,1,0,0,0
+access_crm_lead_scoring_frequency_field_base,access_crm_lead_scoring_frequency_field_base,model_crm_lead_scoring_frequency_field,base.group_system,1,0,0,0

```

Comment:
``access_crm_lead_scoring_frequency_field_system``maybe ? (not sure why you us the 'base' naming, main info is system here)